include/nbl/asset/interchange/IAssetLoader.h

```diff
@@ -95,11 +96,12 @@ class IAssetLoader : public virtual core::IReferenceCounted
 
     struct SAssetLoadParams
     {
-        SAssetLoadParams(	size_t _decryptionKeyLen = 0u, const uint8_t* _decryptionKey = nullptr,
-							E_CACHING_FLAGS _cacheFlags = ECF_CACHE_EVERYTHING,
-							const char* _relativeDir = nullptr, const E_LOADER_PARAMETER_FLAGS& _loaderFlags = ELPF_NONE) :
+		SAssetLoadParams(size_t _decryptionKeyLen = 0u, const uint8_t* _decryptionKey = nullptr,
+			E_CACHING_FLAGS _cacheFlags = ECF_CACHE_EVERYTHING,
+			const char* _relativeDir = nullptr, const E_LOADER_PARAMETER_FLAGS& _loaderFlags = ELPF_NONE, system::logger_opt_ptr&& _logger = nullptr) :
```

Comment:
why r-value reference for a non-smart pointer? (makes no difference, only smart pointer benefits)